src/main/kotlin/com/kotlin/boardproject/domain/notification/service/NotificationServiceImplRDB.kt

```diff
@@ -1,12 +1,12 @@
-package com.kotlin.boardproject.service
+package com.kotlin.boardproject.domain.notification.service
 
-import com.kotlin.boardproject.common.exception.EntityNotFoundException
-import com.kotlin.boardproject.dto.notification.GetNotificationsResponseDto
-import com.kotlin.boardproject.dto.notification.NotificationCreateDto
-import com.kotlin.boardproject.dto.notification.NotificationResponseDto
-import com.kotlin.boardproject.model.Notification
-import com.kotlin.boardproject.repository.NotificationRepository
-import com.kotlin.boardproject.repository.UserRepository
+import com.kotlin.boardproject.global.exception.EntityNotFoundException
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end